ci-operator/jobs/infra-image-mirroring.yaml

```diff
@@ -1,12 +1,18 @@
 periodics:
 - agent: kubernetes
   cluster: app.ci
-  cron: '@hourly'
```

Comment:
This is mysterious :thinking: 
The logs is not hourly but just every 6 hours https://prow.ci.openshift.org/job-history/gs/origin-ci-test/logs/periodic-image-mirroring-knative
But anyway I changes it to daily as every 6 hours or hourly is too frequent.

Reply:
Hm might be because it runs for 5h (which seems to be the max), then its started after an hour, which would result in 6h intervals.
Nevertheless, this job will fail as long as we either:
- Can create more jobs to split eventing, eventig-kafka and serving
- Or cleanup all tag images from eventing on the internal registry (which is the root cause of the long delay). Not sure if we can do this, though.
But adding reporting is great, thank you.

Reply:
@nak3 let's keep `@hourly` here, with the changes to the mirror config coming up, the job should be back to run in minutes.